examples/thermostat/thermostat-common/include/thermostat-delegate-impl.h

```diff
@@ -117,21 +117,21 @@ class ThermostatDelegate : public Delegate
      * invalid index.
      *
      */
-    size_t GetThermostatSuggestionIndexWithEarliestEffectiveTime(uint32_t currentMatterEpochTimestampInSeconds);
+    size_t GetThermostatSuggestionIndexWithEarliestEffectiveTime(System::Clock::Seconds32 currentMatterEpochTimestampInSeconds);
```

Comment:
```suggestion
    size_t GetThermostatSuggestionIndexWithEarliestEffectiveTime(System::Clock::Seconds32 currentMatterEpochTimestamp);
```

You know it's seconds, because it's in the type now.